cli/script/command-executor.ts

```diff
@@ -918,6 +923,73 @@ export var release = (command: cli.IReleaseCommand): Promise<void> => {
         });
 }
 
+export var releaseCordova = (command: cli.IReleaseCordovaCommand): Promise<void> => {
+    var platform: string = command.platform.toLowerCase();
+    var platformFolder: string = path.join(process.cwd(), "platforms", platform);
+    var platformCordova: string = path.join(platformFolder, "cordova");
+    var outputFolder: string;
+    var preparePromise: Promise<void>;
+
+    if (platform === "ios") {
+        outputFolder = path.join(platformFolder, "www");
+    } else if (platform === "android") {
+        outputFolder = path.join(platformFolder, "assets", "www");
+    } else {
+        throw new Error("Platform must be either \"ios\" or \"android\".");
+    }
+
+    try {
+        var tempPrepareContents = `
+        var path = require('path');
+        var ConfigParser = require('cordova-common').ConfigParser;
+        var Api = require('./Api');
+        var projRoot = path.join(process.cwd(), '../../..');
+        var project = {
+            projectConfig: new ConfigParser(path.join(projRoot, 'config.xml')),
+            root: projRoot,
+            locations: {
+                www: path.join(projRoot, 'www')
+            }
+        };
+        var preparer = new Api();
+        preparer.prepare(project);
+        `;
+
+        var prepareOptions: any = { cwd: platformCordova };
+        var prepareProcess = spawnSync("node", ["-e", tempPrepareContents], prepareOptions);
+    } catch (error) {
+        throw new Error("Unable to prepare project. Please ensure that this is a cordova project and that platform " + platform + " was added with cordova platform add " + platform);
+    }
+
+    try {
+        var configString: string = fs.readFileSync(path.join(process.cwd(), "config.xml"), { encoding: "utf8" });
+        var configPromise = parseXml(configString);
+        var releaseCommand: cli.IReleaseCommand = <any>command;
+
+        releaseCommand.package = outputFolder;
+        releaseCommand.type = cli.CommandType.release;
+
+        return configPromise.then((parsedConfig: any) => {
+            var config = parsedConfig.widget;
+
+            var releaseTargetVersion: string;
+            if (command.appStoreVersion) {
+                releaseTargetVersion = command.appStoreVersion;
```

Comment:
I left it this way for now to lower the overall cost if we choose to start reading arbitrary things from config, although I guess we could just short circuit the parse if we don't have any parameters to read from the config.